lmp/jobserv.yml

```diff
@@ -88,6 +88,95 @@ triggers:
         persistent-volumes:
           bitbake: /var/cache/bitbake
 
+  - name: build-release-stable
+    type: git_poller
+    email:
+      users: 'ci-notifications@foundries.io'
+    params:
+      GIT_URL: |
+        https://github.com/foundriesio/lmp-manifest.git
+      GIT_POLL_REFS: |
+        refs/heads/zeus
+      OTA_LITE_TAG: postmerge-stable
+      AKLITE_TAG: promoted-stable
+    runs:
+      # supported images that have OTA
+      - name: supported-{loop}
+        container: hub.foundries.io/lmp-sdk
+        host-tag: amd64-osf
+        loop-on:
+          - param: MACHINE
+            values:
+              - raspberrypi3-64
+        triggers:
+          - name: ota-{loop}
```

Comment:
Do we want to trigger OTAs now or hold off on that?

Reply:
Guess better to hold off for now, let me remove this piece.